api/src/main/java/org/apache/iceberg/ChangelogOperation.java

```diff
@@ -21,5 +21,7 @@
 /** An enum representing possible operations in a changelog. */
 public enum ChangelogOperation {
   INSERT,
-  DELETE
+  DELETE,
+  UPDATE_PREIMAGE,
+  UPDATE_POSTIMAGE
```

Comment:
Open for naming suggestions

Reply:
I also like the idea of a short version of CDC record Type like (I,D, -U, +U). What do you think?
1. Insert(I), indicating an inserted row
2. Delete(D), indicating a deleted row
3. Pre-update(-U), indicating the original value of an updated row
4. Post-update(+U), indicating the new value of an updated row

Reply:
+1

Reply:
How about `UPDATE_BEFORE` `UPDATE_AFTER`，they're a little bit more intuitive to me.

Reply:
I think we can add some doc for the detail, shorter string brings better performance in case we store row kind info into files and performs comparing row by row later.

Reply:
Yes, if we're gonna store it, then i'd prefer a flink like approcach.
```
    /** Insertion operation. */
    INSERT("+I", (byte) 0),
```

Reply:
This PR was updated to use `UPDATE_BEFORE` and `UPDATE_AFTER` so I am resolving this thread. We can't use short versions in Spark as the `changes` table has been already released.